### PR TITLE
fix(doc): syntax of `rustc-check-cfg` instruction

### PR DESCRIPTION
**Pull request by birkenfeld**
Remove the extra colon, and change the example to something that is actually valid.

### CODE REVIEW

**Comment by weihanglo** [comment]
The new two-colons syntax is valid in the current beta and nightly, and will be available in 1.77.0.

The old one-colon syntax is still supported for compatibility reasons.

See <https://github.com/rust-lang/cargo/pull/12201>

**Comment by birkenfeld** [comment]
Oh I see! Sorry. Do you want to keep the other change?

**Comment by weihanglo** [comment]
The stabilization PR of it is <https://github.com/rust-lang/cargo/pull/13571>. I would recommend adding a comment there for doc improvements.

File: src/doc/src/reference/unstable.md [hunk 1]
```diff
@@ -1157,7 +1157,8 @@ You can use the instruction like this:
 
 ```rust,no_run
 // build.rs
-println!("cargo::rustc-check-cfg=cfg(foo, bar)");
```
Comment (Urgau) [inline src/doc/src/reference/unstable.md:1160]: Btw, this is valid; this defines two different cfgs `foo` and `bar`, not one `foo="bar"`.

But I agree this isn't the best example.

**Comment by weihanglo** [comment]
Going to close this as the PR author of #13571 is aware of the example improvement. Thank you all :)

**Pull request closed without merging**